### DISCUSSION TITLE
Prisma doesn't work when using custom output for Prisma client

### DISCUSSION THREAD

**Discussion by dhirennjaypal** [prisma/prisma#22365] [Q&A]
### Bug description

I need to use multiple databases in my application (MySQL and MongoDB) so I am trying to generate different clients for my application by changing output for both clients. but I encountered but that rows are not being inserted in database

### How to reproduce

1. create new next app via create-next-app
2. install prisma and prisma client
3. create schema.prisma like shown
4. prisma generate dev
5. add this inside Home function in app/page.tsx at start of function
```ts
        import { PrismaClient } from '@/prisma/mysql/client'
        export default function Home() {
          let client = new PrismaClient();
	  client.user.create({
		  data: {
			  name: "test"
		  }
	  }); 
	  client.user.findMany().then((res)=>{
		  console.log("=");
		  console.log(res);
	  });
          /* other function logic as it is */
      }
```

when refreshing page/checking database, no rows appear.

### Expected behavior

a new row with id and name=test should appear in database

### Prisma information

```schema.prisma

generator client {
	provider		=	"prisma-client-js"
	output			=	"./mysql/client"
}

datasource db {
	provider		=	"mysql"
	url				=	env("DATABASE_URL")
	relationMode	=	"prisma"
}

model User {
	id		Int		@id @default(autoincrement())
	name	String
}
```

### Environment & setup

- OS: Windows
- Database: MySQL
- Node.js version: v18.17.0

### Prisma Version

```
prisma                  : 5.7.0
@prisma/client          : 5.7.0
Computed binaryTarget   : windows
Operating System        : win32
Architecture            : x64
Node.js                 : v18.17.0
Query Engine (Node-API) : libquery-engine 79fb5193cf0a8fdbef536e4b4a159cad677ab1b9 (at C:\Users\Administrator\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\query_engine-windows.dll.node)
Schema Engine           : schema-engine-cli 79fb5193cf0a8fdbef536e4b4a159cad677ab1b9 (at C:\Users\Administrator\AppData\Roaming\npm\node_modules\prisma\node_modules\@prisma\engines\schema-engine-windows.exe)
Schema Wasm             : @prisma/prisma-schema-wasm 5.7.0-41.79fb5193cf0a8fdbef536e4b4a159cad677ab1b9
Default Engines Hash    : 79fb5193cf0a8fdbef536e4b4a159cad677ab1b9
Studio                  : 0.495.0
```

**Comment by ludralph** [comment]
Hi @dhirennjaypal 👋 

Thank you for raising this question.

Are you getting any errors when you run the query? What does your setup look like for working with multiple databases?

**Reply by dhirennjaypal** [reply]
I still haven't done multiple databases yet, all i did until now is as explained above.

**Reply by ludralph** [reply]
Hi @dhirennjaypal are you using the correct [custom output](https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/generating-prisma-client#using-a-custom-output-path) path for importing the PrismaClient?

**Reply by dhirennjaypal** [reply]
yes, I updated post for more understanding.

**Reply by janpio** [reply]
Is `import { PrismaClient } from '@/prisma/mysql/client'` expected to work as a identifier of a folder? How is your project configured to make that work?

**Comment by janpio** [comment]
I think your `create` is never executed, as you do not `await` or `.then()` the Prisma Client query. Adding `await` before `client.user.create()` should fix that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
